cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1 on 2024-12-16 15:53
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(
+                null=True, upload_to=cinema.models.movie_image_path
```

Comment:
Ensure that `cinema.models.movie_image_path` is correctly defined to format the image path as specified in the task description: `f"{slugify(movie.title)}-{uuid}{ext}"`. This is crucial for saving images with the correct naming convention.